app/models/download.rb

```diff
@@ -90,30 +90,28 @@ def dir
     @tmpdir
   end
 
-  # Utility methods which clean up work data for use in downloads
-
-  def fandoms
-    string = work.fandoms.size > 3 ? "Multifandom" : work.fandoms.string
-    clean(string)
+  # Mimic roughly the default way of producing HTML page titles
+  # @see WorksController#show
+  # @see ApplicationController#get_page_title
```

Comment:
I think you can remove the comment here. Mimicking `ApplicationController#get_page_title` is a starting point but it's not important for us to keep the two implementations totally in sync going forward.

Reply:
I'm not sure if I understand. It is not important because we should eventually have one implementation instead of two implementations? Or because we don't exactly care how the precise formatting of the titles of downloads and online versions may differ, as long as information is not lost?

I think in the former case, some comment would be useful. Maybe a "TODO" comment to ask merging the two?

Reply:
It's ok if they differ, yeah. For the work page we may want to keep the title more concise. For downloads it's fine if we keep more information. On that note, let's keep using `work.fandoms.string` instead of just the first one (`work.fandoms[0].name`) for the else case.

Reply:
That makes sense, thanks. I'll change the part of the code accordingly.